BinaryObjectScanner/Protection/InterLok.cs

```diff
@@ -23,10 +24,14 @@ public class InterLok : IExecutableCheck<PortableExecutable>
             return null;
         }
         
-        private static string GetVersion(string match)
+        private static string GetVersion(string matchMe)
         {
-            match = match.Remove(match.IndexOf(",")).Trim();
-            return match.Substring("InterLok ".Length + match.IndexOf("InterLok"));
+            var match = new Regex(@"(?<=InterLok )(.*?)(?=,)").Match(matchMe);
+            if (match.Success)
+            {
+                return match.Value;
+            }
+            return "(Unknown Version - Please report to us on GitHub)";
```

Comment:
Please add an extra newline above this one.

Reply:
Added newline, although I may have misunderstood where you wanted it.